src/constants/index.ts

```diff
@@ -0,0 +1,4 @@
+export const INIT_STATE = 'electron-redux.INIT_STATE'
+export const INIT_STATE_ASYNC = 'electron-redux.INIT_STATE_ASYNC'
+export const REPLACE_STATE = 'electron-redux.REPLACE_STATE'
+export const ACTION = 'electron-redux.ACTION'
```

Comment:
Just an idea to use an enum here.
```suggestion
export enum ElectronReduxActions {
  INIT_STATE = 'electron-redux.INIT_STATE',
  INIT_STATE_ASYNC = 'electron-redux.INIT_STATE_ASYNC',
  REPLACE_STATE = 'electron-redux.REPLACE_STATE',
  ACTION = 'electron-redux.ACTION',
}
```